flow-sample/src/main/java/com/example/flow/GsonParceler.java

```diff
@@ -43,7 +42,7 @@ public GsonParceler(Gson gson) {
     }
   }
 
-  @Override public Path unwrap(Parcelable parcelable) {
+  @Override public Object unwrap(Parcelable parcelable) {
```

Comment:
Object could be <T> T so great GsonParceler could be reused by third-parties, like I do for Immutables.org: https://gist.github.com/almozavr/2aab84089b23cdf2d99d